item.rb

```diff
@@ -0,0 +1,26 @@
+require 'date'
+
+class Item
+  attr_reader :id
+  attr_accessor :archived, :genre, :publish_date, :author
+
+  def initialize(publish_date, archived: false)
+    @id = Random.rand(1..1000)
+    @publish_date = publish_date
+    @archived = archived
+  end
+
+  def can_be_archived?
+    year = Time.new.year
+    year - @publish_date.year > 10
+  end
+
+  def move_to_archive
+    self.archived = can_be_archived?
+  end
+
+  def add_genre(genre)
+    @genre = genre
+    genre.items.push(self) unless genre.items.include?(self)
+  end
```

Comment:
- [ ] As required for the project and to correctly match the relationship between classes, kindly ensure that the `Item` class has one `Genre`, `Author`, `Label`. Right now, I noticed that you miss both the `Author` and `Label` in the `Item` class. Make sure you add them by following the same logic you use in the `add_genre(genre)` method and `Getter` should be defined to each one(i.e. ``attr_reader ... :genre, :author, :label`` )